modules/mobile/app/models/mobile/v0/enrollment_status.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class EnrollmentStatus < Common::Resource
+      attribute :id, Types::String
+      attribute :application_date, Types::String.optional.default(nil)
+      attribute :enrollment_date, Types::String.optional.default(nil)
+      attribute :preferred_facility, Types::String.optional.default(nil)
+      attribute :parsed_status, Types::String.optional.default(nil)
+      attribute :primary_eligibility, Types::String.optional.default(nil)
+      attribute :can_submit_financial_info, Types::Bool.optional.default(nil)
+
+      def status
```

Comment:
This was a bit of a shot in the dark. The idea is that different statuses may require the mobile app to check back at different frequencies. So we wouldn't necessarily check the status on every login if the user is enrolled because it's unlikely they were un-enrolled and even if they were, this is just used to show a banner.

Reply:
is the idea behind saving the other non-status attributes to the resource but not outputing them in the serializer so just in case we have to come back and expose those fields, then it'll be easier?

Reply:
This is something I considered and felt torn about. Initially, I exposed all of the data in the serializer, but as far as I know, we'll only use the status. I ended up leaving all of the model attributes as a way of making it obvious what data we have, but removed the rest from the serializer. 

I don't know if we have any sort of team philosophy about whether to only expose what's necessary or to expose everything. In general my tendency is to not expose more data than necessary because it's a potential privacy risk. On the other hand, I think it's possible that if I expose all of the data, the front end team may see the available data and decide to use it in other ways, and they probably won't know about this data otherwise. This is why I've been saying that it's hard to design this without knowing if it will be used for things other than the banner. If I knew that it would only be used for the banner, I'd keep it as is. But since our product is being informed by our apis rather than the other way around, maybe it makes sense to just provide everything.

Reply:
@aherzberg what's your preference? add all the data? I'm open to either approach.

Reply:
I'm open to either really. I see the pros and cons of both. Don't think it's worth changing.

Reply:
It's very easy to add them later if we need to.